contribute/labels-of-issues.md

```diff
@@ -0,0 +1,115 @@
+| kind/unit-test | Need/Add Unit test|
+| kind/security  | Security-related concerns  |
+| kind/testing  |  Needs/Adds tests |
+| kind/good-first  |  beginner friendly |
+
+---
+
+## Help
+
+Labels prefixed with *help* implies that issue needs some help from community/users. Further, you can adjust the level of help required, by specifying small, medium, large etc.
+
+| Label | Description |
+|---|---|
+| help/small  | Need small help on docs (beginner level)|
+| help/medium | Need help to fix the code ( Intermediate level) |
+| help/large | Need help on design changes and code review (Expert level) |
+
+---
+
+## Area
+
+Labels prefixed with *area* indicates that which area issue or PR belong. Here, area implies component of the OpenEBS project. It will be easier to sort an issue based on the area of user's interest.
+
+| Label | Description |
+|---|---|
+| area/maya  | Maya Specific  |
+| area/k8s | Kubernetes specific   |
+| area/ci  | Continuous integration specific  |
+| area/high-availability  | High availability  |
+| area/storage | Storage specific  |
+| area/machine-learning  |  Machine learning specific |
+
+---
+
+## Size
+
+Labels prefixed with *size* defines the size of change done in particular PR.
+
+| Label | Description |
+|---|---|
+| size/L | Size of change is large/7 commits probably |
+| size/M | Size of change is medium/5 commits probably |
+| size/S | Size of change is small/3 commits probably |
+| size/XL | Size of change is extra large/10 commits probably |
+| size/XS | Size of change is extra small/1 commit probably |
+| size/XXL | Size of change is very large/15 commits probably |
+
+### NOTE
+
+Please do not club major changes into single PR. It'll be difficult for a reviewer to review a big PR. In that case, reviewer may discard your PR and will ask you create multiple PRs.
+
+---
+
+### Priority
+
+Labels with prefix *priority* defines the severity of the issue. Severity level of an issue can be given as a suffix.
+
+| Label | Description |
+|---|---|
+| priority/0 | Urgent: Must be fixed in the next build. |
+| priority/1 | High: Must be fixed in any of the upcoming builds but should be included in the release. |
+| priority/2 | Medium: Maybe fixed after the release / in the next release. |
+| priority/3 | Low: May or may not be fixed at all. |
+
+---
+
+### Sub-Project
+
+Labels with prefix *repo* points the subproject where given issue needs to be fixed. If users have no idea where to file the issue then they can file a issue in `openebs/openebs` later OpenEBS contributors/authors/owners will tag issue with the appropriate repository using *repo* label.
+
+| Label | Description |
+|---|---|
+| subproject/mayastor | Related to Replicated PV Mayastor and its components |
+| subproject/zfs-localpv | Related to ZFS Local PV and its components |
+| subproject/lvm-localpv | Related to LVM Local PV and its components |
+| subproject/hostpath-localpv |  Related to Hostpath Local PV and its components |
```

Comment:
dynamic-localpv or dynamic-provisioner-localpv?
In case we actually add a driver called hostpath in the future (eg: unified rawfile+dynamic)
```suggestion
| subproject/dynamic-localpv |  Related to Hostpath Local PV and its components |
```

Reply:
dynamic-localpv in that case sounds more appropriate. Done!